package.json

```
@@ -32,7 +32,7 @@
     "expo-application": "^2.2.1",
     "expo-constants": "*",
     "expo-device": "*",
-    "expo-updates": "^0.2.4",
+    "expo-updates": "~0.3.0",
```

```suggestion
    "expo-updates": "~0.3.5",
```

this latest version will be installed anyways, so might as well be explicit 👍 

can you run `yarn` and commit the changes to `yarn.lock`?